setupAndVerifyJDK.md

```diff
@@ -105,3 +105,37 @@ As output you should get something like this:
 ```
     javac 9-ea
 ```
+
```

Comment:
Could you please specify which Operating systems will support this - in theory, MacOSX and Linux.